include/Core/BulkDelete.php

```diff
@@ -348,6 +349,7 @@ private function get_delete_posts_admin_page() {
 		$posts_page->add_module( new DeletePostsByURLModule() );
 		$posts_page->add_module( new DeletePostsByRevisionModule() );
 		$posts_page->add_module( new DeletePostsByStickyPostModule() );
+		$posts_page->add_module( new DeletePostsByCommentsModule() );
```

Comment:
Add this module before Delete by URL module.

Reply:
Fixed.  But still the module appears at the bottom of the page.  Kindly look into it.